pkgs/top-level/all-packages.nix

```diff
@@ -23709,9 +23709,8 @@ with pkgs;
 
     mod_tile = callPackage ../servers/http/apache-modules/mod_tile { };
 
-    mod_wsgi  = self.mod_wsgi2;
-    mod_wsgi2 = callPackage ../servers/http/apache-modules/mod_wsgi { python = python2; ncurses = null; };
-    mod_wsgi3 = callPackage ../servers/http/apache-modules/mod_wsgi { python = python3; };
+    mod_wsgi  = self.mod_wsgi3;
+    mod_wsgi3 = callPackage ../servers/http/apache-modules/mod_wsgi { };
```

Comment:
```suggestion
    mod_wsgi3 = callPackage ../servers/http/apache-modules/mod_wsgi { };
```
os just mod_wsgi

Reply:
Done.

Also had to move the `throw` back to `all-packages`, since `apacheHttpdPackages` attrset is defined there.